src/content/glossary/durable-objects.yaml

```
@@ -7,7 +7,10 @@ entries:
 
   - term: "namespace"
     general_definition: |-
-      A container for a collection of Durable Objects that all share the same Durable Object (class) definition. A single namespace can have (tens of) millions of Durable Objects. Metrics are scoped per namespace.
+      A logical container for a collection of Durable Objects that all share the same Durable Object (class) definition. A single namespace can have (tens of) millions of Durable Objects. Metrics are scoped per namespace.
```

```suggestion
      A logical collection of Durable Objects that all share the same Durable Object (class) definition. A single namespace can have (tens of) millions of Durable Objects. Metrics are scoped per namespace.
```

I'd omit "container", to avoid confusion with the system-level isolation strategy, which is not directly related to DO namespaces... particularly, to avoid confusion with: https://developers.cloudflare.com/containers/ .